### ISSUE TITLE
PS3: Severe sound issue on SNES emulation

### ISSUE THREAD

**Issue by ZerotakerZX** [libretro/snes9x2010#104]
I’m still using 1.0.0.4 ver on my PS3 cause I can’t use SNES core on latest build. Today’s release that claim to fix some issues still have SNES support crippled.

What I mean is, games run fine, but i you try load a state sound go to bin. Like it gets stuck on single sound effect forever upon loading, making games barely playable. I mean one could finish the game without sound, but it won’t be enjoyable. All other functions beside state loading are super fine.

PS3 ver has only Snes9x 2010 core. On vita its quite unstable, but vita also have some alternative cores, while PS3 ver does not.

Can I fix this myself somehow? If not, is the team aware of the issue? If to, is there a hope of it being fixed?

My PS3 is cex fat model running rebug lite firmware

**Comment by retro-wertz** [comment]
please provide details of that is being tested(rom name for example-unless this happens all the time in every rom), and possibly a savestate. some video would help to for someone who is able to diagnose this.

on a similar note, did you try to re-do your savestate? meaning try to create a new savestate using this new version/core and it still happens?

**Comment by ZerotakerZX** [comment]
Yes, I've tried several games (Eeek the Cat, shin nekketsu koha, Jurassric park 2 to name a few) and every one of them have this issue. I can't point when it started exactly, but somewhere around the time when XMB like interface was included.
I've tried several latest releases of RetroArch and every one had this problem. I can't really provide savestate file, since I've rolled back to 1.0.0.4 again to play SNES games. But every time I've tested new version of RetroArch I tried with fresh savestates.

**Comment by retro-wertz** [comment]
lets do hope someone will look at these issues then, since i dont have ps3 for testing and RPCS3 sadly does not work in my ancient computer :(
